### DISCUSSION TITLE
Pro Version Feature Requests

### DISCUSSION THREAD

**Discussion by Tim675** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#127] [Ideas]
Just some feature ideas for a pro version that I'd be happy to pay for:

1) A way to hide the menubar icon since the app has to be always running on m1. I like to keep my menu bar as clean as possible! And would give the feeling that the feature was more built into the OS. 

2) If not a way to hide the icon than an option to show battery percentage in the icon, ie. then I could hide the native battery icon apple provides and just use the al dente icon. 

Thanks!

**Comment by MatthiasKerbl** [comment]
Hi Tim675,
thank you for your ideas. We will definitely do something in this regard.
